how-to/integrate-with-salesforce/client/src/integrations/salesforce/integration-provider.ts

```diff
@@ -45,6 +47,12 @@ export class SalesForceIntegrationProvider implements IntegrationModule<Salesfor
 	 */
 	private static readonly _PROVIDER_ID = "salesforce";
```

Comment:
I have updated the code to disambiguate the providers based on their module.id from manifest, also the module.title is used for labels. Otherwise the Browse, Help, Connect entries are all labelled Salesforce. We also still need to update the module loading code to use url#module.id as a key, otherwise multiple instances will still not load correctly, unless you make a copy of the .js file and load it separately.

Reply:
Also we cant test with SS on a branch as the integrations use the wks version for loading the modules. And you cant add an integration with a custom url, you can only set the customdata part

Reply:
you can use the custom entry and that would let you specify the full integration entry with url path etc.